class ErroCadastroException(Exception):
    def __init__(self):
        super().__init__
